Cartfile.resolved

```diff
@@ -0,0 +1,2 @@
+github "anviking/Decodable" "v0.5.1"
+github "loudmouth/Interstellar-Contentful" "2.1.0"
```

Comment:
@Khaledgarbaya i plan on obviating this dependency as soon as possible. i have some issues with it and don't see much value add except for saving a few lines of code here and there.